src/pages/PreProgram/PreProgram.module.css

```
@@ -530,13 +540,80 @@
+  margin: 12px 0 20px;
+}
+
+.settingsActions {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 12px;
+  padding: 20px 0 8px;
+  width: 100%;
+
+  margin-top: 16px;
```

Stylelint (17.9.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)